README.md

```diff
@@ -102,6 +102,18 @@ install procedure:
    $ make installcheck
 ```
 
+In order to take use the delta debug features, you need to export the path of the corresponding python packages. For example, for a global install, this would resemble (edit for your installed Python version):
```

Comment:
- In order to take use -> In order to use

Reply:
- This should come before `make installcheck`, because tests will fail if PATH is not exported.